lib/MusicBrainz/Server/Data/Search.pm

```diff
@@ -402,12 +402,15 @@ sub schema_fixup
                 my $area = delete $data->{$json_prop};
                 $area->{gid} = $area->{id};
                 $area->{id} = 1;
+                for my $area_prop (qw( type type-id )) {
+                    delete $area->{$area_prop};
```

Comment:
Unless there's a reason the old search didn't output these, why not use `schema_fixup_type` instead of deleting them?

Reply:
The old search server didn't output them. Although I did end up using schema_fixup_type anyways.